packages/cloudflare/README.md

```diff
@@ -6,65 +6,7 @@ Deploy Next.js apps to Cloudflare!
 
 ## Get started
 
-You can use [`create-next-app`](https://nextjs.org/docs/pages/api-reference/cli/create-next-app) to start a new application or take an existing Next.js application and deploy it to Cloudflare using the following few steps:
-
-## Configure your app
-
-- add the following `devDependencies` to the `package.json`:
-
-  ```bash
-  npm add -D wrangler@latest @opennextjs/cloudflare
-  # or
-  pnpm add -D wrangler@latest @opennextjs/cloudflare
-  # or
-  yarn add -D wrangler@latest @opennextjs/cloudflare
-  # or
-  bun add -D wrangler@latest @opennextjs/cloudflare
-  ```
-
-- add a `wrangler.toml` at the root of your project
-
-  ```toml
-  #:schema node_modules/wrangler/config-schema.json
-  name = "<your-app-name>"
-  main = ".open-next/worker.js"
-
-  compatibility_date = "2024-09-23"
-  compatibility_flags = ["nodejs_compat"]
-
-  # Use the new Workers + Assets to host the static frontend files
-  assets = { directory = ".open-next/assets", binding = "ASSETS" }
-  ```
-
-- add a `open-next.config.ts` at the root of your project:
-
-```ts
-import type { OpenNextConfig } from "open-next/types/open-next";
-
-const config: OpenNextConfig = {
-  default: {
-    override: {
-      wrapper: "cloudflare-node",
-      converter: "edge",
-      // Unused implementation
-      incrementalCache: "dummy",
-      tagCache: "dummy",
-      queue: "dummy",
-    },
-  },
-
-  middleware: {
-    external: true,
-    override: {
-      wrapper: "cloudflare-edge",
-      converter: "edge",
-      proxyExternalRequest: "fetch",
-    },
-  },
-};
-
-export default config;
-```
+To get started with the adapter visit the [official get started documentation](https://opennext.js.org/cloudflare/get-started).
```

Comment:
Can you please also drop the KI section below which is out of date

Reply:
done [da78a85](https://github.com/opennextjs/opennextjs-cloudflare/pull/265/commits/da78a853aaadb46feae197ed90c0274d5c4624d9) 👍